gunicorn/arbiter.py

```diff
@@ -237,10 +249,12 @@ def run(self):
                 self.pidfile.unlink()
             sys.exit(-1)
 
-    def handle_chld(self, sig, frame):
+    def handle_cld(self):
```

Comment:
`SIGNALS` installs only `SIGCHLD`, not `SIGCLD`. What is this alias for?

Reply:
Heh, this is a good question, I had to dig a little bit :)

Short answer: Because self.SIG_NAMES[Signals.SIGCHLD] => 'cld'.

Long answer: Consider the following code:

```python
import signal

# I love dynamic languages
SIGNALS = [getattr(signal, "SIG%s" % x)
               for x in "CHLD HUP QUIT INT TERM TTIN TTOU USR1 USR2 WINCH".split()]
SIG_NAMES = dict(
    (getattr(signal, name), name[3:].lower()) for name in dir(signal)
    if name[:3] == "SIG" and name[3] != "_"
)
```
While SIGNALS looks correct, the bug is inside SIG_NAMES, specifically how we loop over `dir(signal)`. On Linux, this list of attributes contains both the strings 'SIGCHLD' and 'SIGCLD' in this order. So what happens? Well, the same thing as `{ 'foo': 1, 'foo': 2 }`: we end up with `{ 'foo': 2 }`. 

One way to fix this would be to loop over `[s.name for s in signal.Signals]` instead, which doesn't seem to include both SIGCLD and SIGCHLD (only the latter is included).